sycl/include/sycl/ext/intel/experimental/esimd/simd_view.hpp

```diff
@@ -68,6 +68,64 @@ class simd_view : public detail::simd_view_impl<BaseTy, RegionTy,
     return *this;
   }
 
+#define DEF_BINOP(BINOP, OPASSIGN)                                             \
```

Comment:
Out of curiosity: 'OPASSIGN' seems unused in the macro definition. Is it planned to add operator OPASSIGN later?

Reply:
> Out of curiosity: 'OPASSIGN' seems unused in the macro definition. Is it planned to add operator OPASSIGN later?

You're right. It is unused and can be deleted. It is a copy-paste leftover.